tests/common.go

```diff
@@ -1510,6 +1546,9 @@ func ScheduleAppsInTopologyEnabledCluster(
 func ValidateApplicationsPurePxctl(contexts []*scheduler.Context) {
 	Step("validate applications", func() {
 		for _, ctx := range contexts {
+			if ctx.App.Key == "nginx-fa-davol" {
```

Comment:
Why did we need to skip volume validation for this app? The hard-coded checks here really worry me so I'd like to remove them if we can.

Reply:
Hi Adam, thank you so much for above comments. We created nginx-fa-davol app for FADA volume & we have wordpress and nginx-withiout-enc apps for FBDA volumes, we were facing some issues in executing clone test from all test of PureSDK with new nginx-da-vol hence we were skipping the volume validation.
But after looking your comments we thought there is only way to avoid hard coded checks to run all Pure tests against our new app nginx-fa-davol along with validateMountOptions tests. So we debugged issue, fixed it for new app and updated the PR with same.